radio/src/targets/horus/CMakeLists.txt

```diff
@@ -162,21 +151,26 @@ elseif (PCB STREQUAL X12S)
   set(BLUETOOTH ON)
   set(PCBREV "13" CACHE STRING "PCB Revision")
   if(${PCBREV} GREATER 10)
-    option(INTERNAL_GPS "Internal GPS installed" YES)
+    option(INTERNAL_GPS "Support for internal GPS" YES)
   else()
-    option(INTERNAL_GPS "Internal GPS installed" NO)
+    option(INTERNAL_GPS "Support for internal GPS" NO)
     if(NOT INTERNAL_GPS)
       message("Horus: Internal GPS is optional, use INTERNAL_GPS=YES option to enable it")
     endif()
   endif()
-  set(INTERNAL_GPS_BAUDRATE "9600" CACHE STRING "Baud rate for internal GPS")
   set(AUX2_SERIAL ON) # wire to GPS
   set(USE_VS1053B ON)
   set(USE_ADS79xx ON)
   set(FONTS_TARGET x12s_fonts)
   add_definitions(-DMANUFACTURER_FRSKY)
 endif()
 
+# enable GPS support unless otherwise disabled
+if(NOT (AUX_SERIAL STREQUAL "OFF"))
```

Comment:
I think you meant: if(AUX_SERIAL)

Reply:
True... no need for a double negative! lol